CONTRIBUTING.md

```diff
@@ -31,45 +38,56 @@ Ready to contribute? Here's how to set up `simulariumio` for local development.
     Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass linting and
-   tests, including testing other Python versions with make:
+   tests with [just](https://github.com/casey/just):
```

Comment:
`just` is _just_ a better `make`. Fancy :sparkles: written in Rust. But the real advantages imo are two:
1. variable handling is WAY easier and it has custom functions we can call for certain things like `{{justfile_directory()}}` to get the directory of the `Justfile` itself or `{{join()}}` for path joining on all OSes.
2. It runs from any subdir! `make` must be run in the dir where the `Makefile` is, but `just` will search up directories until it finds one.